modules/ROOT/pages/first-look/linux-first-look.adoc

```diff
@@ -30,25 +30,25 @@ See link:https://neo4j.com/docs/operations-manual/current/installation/linux/tar
 
 == Server installation
 === Unpack NOM server from Neo4j package
-   
+
 [source, terminal]
 ----
    tar -xzf NEO4J_HOME/products/neo4j-ops-manager-server-*-unix.tar.gz
 ----
-    
+
 Extracted directory is now known as NOM_SERVER_HOME.
 
 === Generate self-signed certificates
 
 Generate `server.cer` and `server.pfx` as described *xref:installation/self-signed-certificate.adoc[here]*.
 Use the password applied for the certificate store instead of `<PASSWORD>` in the next section.
 
-=== Run NOM server as console application 
+=== Run NOM server as console application
 
 [source, terminal]
 ----
     java -jar ./lib/server.jar \
-        --spring.neo4j.uri=neo4j: neo4j://localhost:7687\
+        --spring.neo4j.uri=neo4j:neo4j://localhost:7687\
```

Comment:
This seems to be incorrect still.
Should be
`--spring.neo4j.uri=neo4j://localhost:7687\`